src/WebPages/PortletFramework/DropDownPartField.cs

```diff
@@ -82,13 +82,11 @@ private void BuildList()
             if (this.DropdownOptions.CommonType == DropDownCommonType.ContentTypeDropdown)
             {
                 // special use-case, content type list is defined in webconfig
-                NodeQueryResult contentTypeNames = GetWebContentTypeList();
+                var contentTypeNames = GetWebContentTypeList();
                 if (contentTypeNames.Count > 0)
                 {
-                    foreach (var ctContent in contentTypeNames.Nodes.Select(Content.Create))
-                    {
+                    foreach (var ctContent in contentTypeNames.Nodes)
```

Comment:
Please wrap nodes into Content objects, otherwise _DisplayName_ will return _string resources_. Localization happens in the Content layer.